src/h2olog/h2olog.cc

```diff
@@ -39,11 +41,18 @@ static void usage(void)
     printf(R"(h2olog (h2o v%s)
 Usage: h2olog -p PID
        h2olog quic -p PID
-       h2olog quic -s response_header_name -p PID
-Other options:
-    -h Print this help and exit
+Optional arguments:
     -d Print debugging information (-dd shows more)
+    -h Print this help and exit
+    -l Print the list of handled USDTs and exit
+    -s RESPONSE_HEADER_NAMES Response header names to show, e.g. "content-type"
+    -t EVENT_TYPES Fully-qualified probe names to show, e.g. "quicly:accept"
```

Comment:
Assuming that the intent is to specify "probes", as suggested in the comment, I think the name of the option character should better be `-p` and that the explanation should be `-p probe_name`.

Reply:
I agree that `-t` for "event type" is a little bit odd, but unfortunately `-p` is already used for "PID". `-p PID` is used by lots of BCC tools, so we'd better to keep `-p` as is.

So how about `-u` for "USDT"?

Reply:
Ah thank you for reminding me of `-p`. Then, how about `-t` for "tracepoint"? The fact that we use USDT is an implementation detail, IMO.

Reply:
Nice! "tracepoint" sounds good to me!